config/plugin/windows_defender_problem.ps1

```diff
@@ -0,0 +1,12 @@
+# This plugin checks to see if windows defender detects any threats to the node.
+
+$OK=0
+$NONOK=1
+
+$windowsDefenderThreats = Get-MpThreatDetection
+
+if ($windowsDefenderThreats.length -ne 0) {
+    exit $NONOK
```

Comment:
I think it's clear to just return 0 vs 1 here. Aliasing the variables adds unnecessary indirection.

Reply:
Is it common knowledge that 1 indicates a bad exit, and 0 is an OK exit? I haven't had to deal with exits before. I was thinking that the variables makes it easy to read and understand what it indicates, but if it's common knowledge then I am good to remove them and return just 0 or 1.

Reply:
exit code 0 indicates success thats common